src/responses/commands.ts

```diff
@@ -48,6 +49,32 @@ export const uptime = new Response(
   "I tell you how long I've been awake :timer:"
 )
 
+const picPhrase = RegExp(`${CMD_PREFIX}.*`)
```

Comment:
Shouldn't you be searching for $pic here? This regex matches on all commands.

Reply:
Oops.